imap_processing/tests/cdf/test_utils.py

```diff
@@ -32,11 +35,11 @@ def test_dataset():
                 ],
             )
         },
-        attrs=swe_l1a_global_attrs.output()
+        attrs=swe_attrs.get_global_attributes("imap_swe_l1a_sci")
         | {
-            "Logical_source": "imap_swe_l1_sci",
+            "Logical_source": "imap_swe_l1a_sci",
             "Data_version": "001",
-            "Logical_file_id": "imap_swe_l1_sci_20100101_v001",
+            "Logical_file_id": "imap_swe_l1a_sci_20100101_v001",
         },
```

Comment:
Aren't some (all?) of these already set in the `get_global_attributes()` yaml definitions now? So you may not need to update this anymore.

Reply:
`Logical_file_id` is not still. But good point about removing this `or` option.